src/views/ClusterViz/styles.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../stylesheets/base';
+
+.cluster {
+    @extend %main-content;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .header {
+        @include background-color($color-background-header);
+        display: flex;
+        align-items: center;
+        flex-shrink: 0;
+        justify-content: space-between;
+        border-bottom: $width-separator-thin solid $color-separator-alt;
+        padding: $spacing-small;
+    }
+
+    .container {
+        display: flex;
+        position: relative;
+        flex-grow: 1;
+
+        .forced-directed-graph {
+            flex-grow: 1;
+            height: auto;
+            overflow: auto;
+        }
+
+        .tables {
+            flex-basis: 320px;
```

Comment:
Create a variable at the top of page and use it, do not declare sizes directly in stylings.